cytodynamics-nucleus/src/main/java/com/linkedin/cytodynamics/nucleus/IsolatingClassLoader.java

```diff
@@ -175,8 +176,19 @@ public URL getResource(String name) {
       if (!isBlacklisted) {
         delegateClass = delegateRelationship.getDelegateClassLoader().loadClass(name);
```

Comment:
Since you are at this, would you mind modifying the return java docs to use `delegateRelationship` instead of `parentRelationship`?